glide.yaml

```diff
@@ -3,12 +3,17 @@ import:
 - package: github.com/cenkalti/backoff
   version: v1.0.0
 - package: github.com/giantswarm/microkit
+  version: master
   subpackages:
   - error
   - logger
 - package: github.com/juju/errgo
+  version: master
 - package: k8s.io/client-go
-  version: v3.0.0-beta.0
+  version: v3.0.0
```

Comment:
`-beta.0` is dropped. There is also `v4.0.0-beta.0`, but I don't want to deal with it ATM. Probably introduces CRD support.